habana/pytorch/buildspec-1-10-1.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
We have made some changes in the way we store the older buildspec files. Can you please refer to the new naming convention?

Reply:
I'll just delete the build-1-10.yml and rename the buildspec-1-10-1.yml to buildspec-1-10.yml. I don't want to change the naming convention.